bin/cqlsh.py

```diff
@@ -1061,7 +1061,7 @@ def do_select(self, parsed):
         tracing_was_enabled = self.tracing_enabled
         ksname = parsed.get_binding('ksname')
         stop_tracing = ksname == 'system_traces' or (ksname is None and self.current_keyspace == 'system_traces')
-        self.tracing_enabled = self.tracing_enabled and not stop_tracing
+        self.tracing_enabled = "off" if stop_tracing else self.tracing_enabled
```

Comment:
Nit: this probably should be changed to "tracing_state" (or some other non-binary name) at some point given the nature of this change.  Hardly essential though.

Reply:
changed to `tracing_style`